### ISSUE TITLE
Site URL still showing on home screen even after patching remove general ads patch.

### ISSUE THREAD

**Issue by HarshadDhokale** [revanced/revanced-patches#959]
### Type

Cosmetic

### Bug description

![Screenshot_20221103-070555_YouTube](https://user-images.githubusercontent.com/105629462/199633321-68cbfce9-520b-4be5-9ac8-301f52b5f002.png)

### Steps to reproduce

Just patch and surf and you will notice it

### Relevant log output

```shell
Initializing installer
Creating working directory
Copying original apk
Unpacking input apk
Reading dex files
Decoding AndroidManifest.xml only, because resources are not needed
Merging integrations
Deleting existing resource cache directory
Decoding resources
Executing all patches
Executing 'resource-mapping' of type: resource
Executing 'integrations' of type: bytecode
Skipping 'integrations' because it has already been applied
Executing 'locale-config-fix' of type: resource
Skipping 'resource-mapping' because it has already been applied
Executing 'settings-resource-patch' of type: resource
Executing 'settings' of type: bytecode
Skipping 'locale-config-fix' because it has already been applied
Executing 'general-resource-ads' of type: resource
Executing 'general-ads' of type: bytecode
Applied general-ads
Skipping 'integrations' because it has already been applied
Skipping 'settings' because it has already been applied
Skipping 'integrations' because it has already been applied
Skipping 'integrations' because it has already been applied
Executing 'player-controller-hook' of type: bytecode
Skipping 'settings' because it has already been applied
Skipping 'integrations' because it has already been applied
Executing 'video-id-hook' of type: bytecode
Executing 'fix-playback' of type: resource
Executing 'video-ads' of type: bytecode
Applied video-ads
Skipping 'locale-config-fix' because it has already been applied
Executing 'bottom-controls-resource-patch' of type: resource
Skipping 'locale-config-fix' because it has already been applied
Skipping 'settings' because it has already been applied
Executing 'downloads-resource-patch' of type: resource
Skipping 'resource-mapping' because it has already been applied
Executing 'player-controls-bytecode-patch' of type: bytecode
Skipping 'video-id-hook' because it has already been applied
Executing 'downloads' of type: bytecode
Applied downloads
Skipping 'integrations' because it has already been applied
Skipping 'settings' because it has already been applied
Executing 'seekbar-tapping' of type: bytecode
Applied seekbar-tapping
Skipping 'integrations' because it has already been applied
Skipping 'integrations' because it has already been applied
Executing 'player-type-hook' of type: bytecode
Skipping 'settings' because it has already been applied
Executing 'swipe-controls-resource-patch' of type: resource
Executing 'swipe-controls' of type: bytecode
Applied swipe-controls
Skipping 'locale-config-fix' because it has already been applied
Executing 'premium-heading' of type: resource
Applied premium-heading
Skipping 'integrations' because it has already been applied
Skipping 'resource-mapping' because it has already been applied
Skipping 'settings' because it has already been applied
Executing 'hide-create-button' of type: bytecode
Applied hide-create-button
Skipping 'integrations' because it has already been applied
Skipping 'video-id-hook' because it has already been applied
Skipping 'locale-config-fix' because it has already been applied
Skipping 'settings' because it has already been applied
Executing 'return-youtube-dislike-resource-patch' of type: resource
Executing 'return-youtube-dislike' of type: bytecode
Applied return-youtube-dislike
Skipping 'player-controller-hook' because it has already been applied
Skipping 'player-controls-bytecode-patch' because it has already been applied
Skipping 'integrations' because it has already been applied
Skipping 'locale-config-fix' because it has already been applied
Skipping 'settings' because it has already been applied
Skipping 'resource-mapping' because it has already been applied
Executing 'sponsorblock-resource-patch' of type: resource
Skipping 'video-id-hook' because it has already been applied
Executing 'sponsorblock' of type: bytecode
Applied sponsorblock
Executing 'litho-components-theme' of type: bytecode
Skipping 'locale-config-fix' because it has already been applied
Executing 'theme' of type: resource
Applied theme
Executing 'client-spoof' of type: bytecode
Applied client-spoof
Skipping 'integrations' because it has already been applied
Skipping 'resource-mapping' because it has already been applied
Executing 'custom-playback-speed' of type: bytecode
Applied custom-playback-speed
Skipping 'integrations' because it has already been applied
Skipping 'settings' because it has already been applied
Executing 'minimized-playback' of type: bytecode
Applied minimized-playback
Skipping 'settings' because it has already been applied
Applied settings
Skipping 'settings' because it has already been applied
Executing 'custom-video-buffer' of type: bytecode
Applied custom-video-buffer
Repacking patched apk
Compiling resources
Creating new dex file
Writing modified dex files
Finished!
```

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by shekhawat2** [comment]
+1
![Screenshot_20221103-170251_YouTube](https://user-images.githubusercontent.com/20217523/199710484-bd085732-c6fa-4197-8f90-bb2f3c839c8f.png)
![Screenshot_20221103-170311_YouTube](https://user-images.githubusercontent.com/20217523/199710533-cfd0de3f-e3a6-4b3b-b04f-aac525fc048e.png)

**Comment by nitinnagar23** [comment]
I'm also facing the same bug, Pull down to refresh feed is working. Seems like there are some bugs in ads patches.

![Screenshot_2022-11-04-14-45-41-782_app revanced android youtube](https://user-images.githubusercontent.com/57346763/199936696-9a0408fe-98e7-4eed-b6cd-f616c8fc19df.jpg)

**Comment by Aurobindoji** [comment]
Noticed this bug in ad patch.
Top ribbon of ad.

![Screenshot_2022-11-10-18-34-57-039_app.revanced.android.youtube.jpg](https://user-images.githubusercontent.com/114210081/201100570-f3b1a475-c92e-47ac-9a65-f07f9e0e01f0.jpg)

**Comment by l-Nuril-l** [comment]
![Screenshot_2022-11-10-20-08-26-752_app revanced android youtube](https://user-images.githubusercontent.com/49325887/201174626-89e3e87a-c0f0-47ec-9ec6-9084d2985732.jpg)
In my case, this problem appears under the video itself.

**Comment by l-Nuril-l** [comment]
Fixed

**Comment by smonsec** [comment]
Faced this issue too. Even few video timeline ads are also present here and there despite the ads patch. I'm using 17.43.36 version.

![Screenshot_20221118-151631_YouTube_ReVanced_1.png](https://user-images.githubusercontent.com/68259771/202761356-36d5dab4-3164-40cd-9dc1-c635ce8d4bc2.png)

**Comment by oSumAtrIX** [comment]
Fixed in 97fd3fda79ff4bb2fadd0e6c4aa80b96d71f46da.
